app/actions/remote/team.ts

```diff
@@ -98,6 +98,46 @@ export async function addUserToTeam(serverUrl: string, teamId: string, userId: s
     }
 }
 
+export async function addUsersToTeam(serverUrl: string, teamId: string, userIds: string[]) {
+    let client;
+    try {
+        client = NetworkManager.getClient(serverUrl);
```

Comment:
I guess this can go in the same try/catch as the rest

Reply:
Done